docker-compose.yml

```diff
@@ -66,3 +67,9 @@ services:
       - keycloak
     volumes:
       - ./:/var/www/html:delegated
+
+  maildev:
+    image: 'maildev/maildev'
+    ports:
+      - "8002:1080"
```

Comment:
```suggestion
      - "8003:1080"
```

Reply:
@boubkerbribri I think, it can fix.

Reply:
Why not use 1080 ? (FYI 8002 port is used with ssl on all our tools, ex https://github.com/PrestaShop/PrestaShop/blob/develop/.github/workflows/cron_nightly_tests_reusable.yml#L171)

Reply:
I'm voting for `1080` just because we are also using 1025 for the smtp port.

Maybe this should be decided on "mangement" level if the ports should be bunched up in the 80xx range or not.